src/preload.js

```diff
@@ -20,7 +20,7 @@ const PRELOAD_PLAYERS = [
   {
     Player: Vimeo,
     configKey: 'vimeo',
-    url: 'https://vimeo.com/127250231'
+    url: 'https://vimeo.com/261464690'
```

Comment:
I've uploaded a silent video to Vimeo. Let's use that:

```suggestion
    url: 'https://vimeo.com/300970506'
```

Reply:
Thanks, updated !